### PR TITLE
Add Azure identity sample for Event Hubs README

### PR DESCRIPTION
**Pull request by conniey**
* Adds an Azure identity sample using ClientSecretCredential
* Changes the absolute links to relative
* Organizes the links at the bottom.

### CODE REVIEW

File: eventhubs/client/README.md [hunk 1]
```diff
@@ -40,26 +40,62 @@ documentation][event_hubs_product_docs]
...
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.0-preview.1</version>
+</dependency>
+```
+
+All the implemented ways to get a credential can be found under the `com.azure.identity.credential` package. The sample below shows how to use an Azure Active Directory (AAD) application client secret to authorize with Azure Event Hubs.
```
Comment (jsquire) [inline eventhubs/client/README.md]: nit:  "get a credential" sounds a bit awkward to me.  Maybe consider "request a credential" or similar?

File: eventhubs/client/README.md [hunk 2]
```diff
@@ -40,26 +40,62 @@ documentation][event_hubs_product_docs]
...
+
+#### Authorizing with AAD application client secret
```
Comment (jsquire) [inline eventhubs/client/README.md]: nit:  did you mean for this to have 3 "#"?

Comment (conniey) [reply eventhubs/client/README.md]: I meant to have 4. The 3rd level header is "Create an Event Hub client using Microsoft identity platform (formerly Azure Active Directory)", and this next section builds off of that previous one.

File: eventhubs/client/README.md (hunk 1 above)
Comment (conniey) [reply eventhubs/client/README.md]: Thanks! fixed.

**Pull request merged**